src/components/resources/discoveryAndLoadbalancing/services/ServiceDetails.tsx

```diff
@@ -47,7 +47,7 @@ const ServiceDetails: React.FunctionComponent<IServiceDetailsProps> = ({ item, t
                 return (
                   <IonChip key={index} className="unset-chip-height">
                     <IonLabel>
-                      {port.name} {port.port}
+                      {port.name} {port.port} &nbsp;
                       {port.nodePort}/{port.protocol} &gt; {port.targetPort}
```

Comment:
I looked at the type definition for the fields and `name`, `nodePort`, `protocol` and `targetPort` are optional. Can you change this to:

```suggestion
                      {port.name ? `${port.name} ` : ''}
                      {port.port}
                      {port.nodePort ? ` ${port.nodePort}` : ''}
                      {port.protocol ? `/${port.protocol}` : ''}
                      {port.targetPort ? ` > ${port.targetPort}` : ''}
```

This wouldn't force a space between `port` and `/protocol` if there isn't a `nodePort`.